UefiPayloadPkg/SPI/stopwatch.h

```diff
@@ -0,0 +1,219 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
Delete this file i think. It would be easier to use delay function from here UefiPayloadPkg/Library/AcpiTimerLib/AcpiTimerLib.c

Reply:
@IgorBagnucki

Reply:
Deleted